### DISCUSSION TITLE
I suggest adding an option to disable the auto-sorting of the open scripts list in the editor.

### DISCUSSION THREAD

**Discussion by boblako** [godotengine/godot-proposals#11128] [Editor]
1. I move Model.gd to the top of the list to find the script quickly by sight.
2. I open the accept_moves script using ctrl + left click.
3. Model.gd jumps to the middle of the list because the entire list sorts itself automatically.

Solution 1:  
Add an option to disable the automatic sorting of the scripts list. New scripts would be added to the top of the list without sorting.

Solution 2:  
Add an option to highlight scripts with color to make them easier to locate in the list.

https://github.com/user-attachments/assets/035574ed-0cf9-4ef3-b7f1-3fe1146059c8

![colored script names](https://github.com/user-attachments/assets/6a550c80-3d5f-425f-98a6-723cb1316a8f)

**Comment by Jesusemora** [comment]
Yes! So much yes!

I would like the editor to remember where I put the scripts so inherited classes or  similar appear next to each other for quick switching.

But this would also help. I get lost wherever the scripts are automatically moved and have to scroll to find them, so have to resort to Ctrl clicking and clicking the script button next to the node to find them.

**Comment by Mickeon** [comment]
Are you looking for this?
![image](https://github.com/user-attachments/assets/791ab530-cd12-4a4a-9537-0a02e62b6883)

**Reply by Jesusemora** [reply]
Yes, I've been looking for this for months.
Found it in **editor settings** in case anyone else is looking for it.

It's very hidden, I would still like this to be in the actual script editor next to the sort methods button because it makes no sense to have them separate.
